processor/elasticinframetricsprocessor/testdata/k8smetrics_drop_original/output-metrics.yaml

```diff
@@ -0,0 +1,117 @@
+resourceMetrics:
+  - resource:
+      attributes:
+        - key: k8s.pod.name
+          value:
+            stringValue: test-pod
+    scopeMetrics:
+      - metrics:
+          - gauge:
+              dataPoints:
+                - asDouble: 345
+                  attributes:
+                    - key: data_stream.dataset
+                      value:
+                        stringValue: kubernetes.pod
+                    - key: event.dataset
+                      value:
+                        stringValue: kubernetes.pod
+                    - key: otel_remapped
+                      value:
+                        boolValue: true
+                    - key: service.type
+                      value:
+                        stringValue: kubernetes
+                  timeUnixNano: "1000000"
+            name: kubernetes.pod.cpu.usage.limit.pct
+          - gauge:
+              dataPoints:
+                - asDouble: 0
```

Comment:
Are those `0`s here on purpose? Seems a bit odd when the input file provides the `k8s.pod.cpu_limit_utilization` equal to `345` to have additional metrics reported with zero values.

Reply:
Ah that's exactly what https://github.com/elastic/opentelemetry-collector-components/pull/147#issuecomment-2386081423 reports 👍🏽